Libraries/ReactNative/NativeUIManager.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
```

Comment:
Can you use strict-local and update references to Object which is implicit any?

Reply:
@thymikee object is a desired return result according to the issue filed. In codegen, I think it has a different meaning from any.

Reply:
I tend to use `{[key: string]: mixed}` instead of Object. I'm not sure what assumptions codegen is making, but I would expect it (sooner or later) to get data from Flow Type AST

Reply:
For now, let's stick with `Object`. The internal codegen isn't very smart yet, so we can improve it over time later.